rail_manipulation_msgs/SegmentedObject segmented_object
---